scipy/stats/_distn_infrastructure.py

```diff
@@ -2559,12 +2560,26 @@ def fit(self, data, *args, **kwds):
 
         >>> from scipy.stats import beta
         >>> a, b = 1., 2.
-        >>> x = beta.rvs(a, b, size=1000)
+        >>> x = beta.rvs(a, b, size=1000, random_state=80)
 
         Now we can fit all four parameters (``a``, ``b``, ``loc`` and
         ``scale``):
 
         >>> a1, b1, loc1, scale1 = beta.fit(x)
+        >>> a1, b1, loc1, scale1
+        (0.998314523745146, 2.0351090001496326, 0.0004988689720895926, 1.0059326261917274)
+
+        The fit can be done also using a custom optimizer:
+
+        >>> from scipy.optimize import minimize
+        >>> def custom_optimizer(func, x0, args=(), disp=0):
+        ...     res = minimize(func, x0, args, method="COBYLA", options={"disp": disp})
```

Comment:
```suggestion
        ...     res = minimize(func, x0, args, method="slsqp", options={"disp": disp})
```
Let's avoid recommending use of COBYLA. SLSQP seems to be much more reliable.